apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -439,7 +439,7 @@ export const getReadyToPublishDocuments = async ({ params: { id }, body }, respo
 		caseId: +id
 	});
 
-	const documentsCount = await documentRepository.getDocumentsCountInByPublishStatus();
+	const documentsCount = await documentRepository.getDocumentsCountInByPublishStatus(+id);
```

Comment:
nit-pick: I always suggest using `Number(id)` instead. Just seems less magic 😄

Reply:
Makes sense! I just copied the unary operator from line 439 tbh 😆